### PR TITLE
feat(tier4_*_launch): move perception modules to simulator.launch.xml

### PR DESCRIPTION
**Pull request by takayuki5168**
Signed-off-by: Takayuki Murooka <takayuki5168@gmail.com>

## Related Issue(required)

https://github.com/autowarefoundation/autoware.universe/issues/455

## Description(required)

scenario sim will publish detection results although currently publish prediction results.
https://github.com/tier4/scenario_simulator_v2/pull/712

We do not call tracking, prediction modules with scenario sim now, so I modified to call these modules with not only simple psim but also scenario sim.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by taikitanaka3** [comment]
@takayuki5168 
can you refer pure perception launch and remove duplicated and unused tier4_perception launch like this? cc: @yukkysaito 
https://github.com/autowarefoundation/autoware.universe/pull/454

File: launch/tier4_simulator_launch/launch/simulator.launch.xml [hunk 1]
```diff
@@ -25,6 +26,47 @@
...
+      </include>
+    </group>
+
+    <!-- traffic light module -->
+    <group if="$(var perception/enable_traffic_light)">
+      <push-ros-namespace namespace="traffic_light_recognition"/>
+      <include file="$(find-pkg-share tier4_perception_launch)/launch/traffic_light_recognition/traffic_light.launch.xml"></include>
+    </group>
+  </group>
+
```
Comment (yukkysaito) [inline launch/tier4_simulator_launch/launch/simulator.launch.xml:69]: The only things that are simulated are detection and obstacle segmentation. Other than that, tracking and prediction are working as usual (no simulation), so should it is put them in simulation.launch?

Comment (takayuki5168) [reply launch/tier4_simulator_launch/launch/simulator.launch.xml:69]: @yukkysaito 
In planning_simulator.launch.xml, perception launch is not called (planning/control launches are called).
Currently, we call perception launch (tracking, prediction, obstacle segmentation) from dummy_perception_publisher when using simple psim.

Now we want to use this perception launch with scenario sim as well, which does not depend on dummy_perception_publisher, so I moved them to simulator.launch.xml.
I'm ok to move to planning_simulator.launch.xml if you want

Comment (yukkysaito) [reply launch/tier4_simulator_launch/launch/simulator.launch.xml:69]: >In planning_simulator.launch.xml, perception launch is not called (planning/control launches are called).
Currently, we call perception launch (tracking, prediction, obstacle segmentation) from dummy_perception_publisher when using simple psim.

Yes. I know :+1: 

What I mean is that tracking and prediction are not simulated. What is being simulated is the point cloud, the vehicle model, and detection.
There is no problem that point cloud, the vehicle model, and detection. are in simulation.launch. 
But there are things in simulation.launch that are not being simulated.
I'm not sure what is best. :thinking:

**Comment by takayuki5168** [comment]
@tkimura4 
I confirmed simple psim works with tier4_autoware_launch

**Pull request closed without merging**